packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -46,5 +46,9 @@ export const DefaultPalette: IPalette = {
   tealLight: '#00b294',
   greenDark: '#004b1c',
   green: '#107c10',
-  greenLight: '#bad80a'
+  greenLight: '#bad80a',
+  contrastBlackDisabled: '#00ff00',
```

Comment:
I have a question - should we be using 'greyText' in all HC modes ? As in, whenever we mention a color for anything, should we mention three sets of colors - one each for normal, hc white on black, hc black on white.